src/SprykerEco/Shared/Payone/Transfer/payone.transfer.xml

```diff
@@ -462,12 +462,15 @@
         <property name="streetNumber" type="string" />
         <property name="zip" type="string" />
         <property name="city" type="string" />
+        <property name="personstatus" type="string" />
```

Comment:
If the lower case is not required in request/response please change it to camelCase otherwise leave it as is.